### PR TITLE
Feat: Added Input&Image-Component

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Added Image and Input Component. Also refactored Styling of Button & Card Component. Components are slightly increased in size. Please refer screenshots or deploy link for visuals.

image
image

### CODE REVIEW

**Review by arjun-computer-geek — COMMENTED** [review]
hey bro I think you should always push different components with different branches. Please don't merge two into one. By the way all looks good for me.

File: components/Button/button.css [hunk 1]
```diff
@@ -66,55 +66,67 @@
...
-    font-size: 1.25rem;
-    height: 2.5rem;
+    font-size: 1.7rem;
+    height: 3.5rem;
     min-width: 3.5rem;
     max-width: max-content;
-    padding-inline-start: 0.75rem;
-    padding-inline-end: 0.75rem;
+    padding-inline-start: 1.25rem;
+    padding-inline-end: 1.25rem;
```
Comment (venkateshwarans) [inline components/Button/button.css:74]: any specific reason to use `padding-inline-start`, `padding-inline-end` and not `padding-left` , `padding-right`?

**Comment by ShekharDhangar** [comment]
> hey bro I think you should always push different components with different branches. Please don't merge two into one. By the way all looks good for me.

Ok, Understood. Won't repeat this.

File: components/Button/button.css (hunk 1 above)
Comment (ShekharDhangar) [reply components/Button/button.css:74]: Its just a short-hand property. No specific reason

Comment (venkateshwarans) [reply components/Button/button.css:74]: I think there is a slight difference between padding and padding-inline - https://css-tricks.com/almanac/properties/p/padding-inline/#aa-values . But if you are aware of it, all good.

Comment (ShekharDhangar) [reply components/Button/button.css:74]: Yes I know the difference. Thankyou for pointing out.

**Pull request merged**